docs/best-practices/routing/routing.md

```diff
@@ -2,13 +2,13 @@
 
 ## Routing Mode
 
-First, you should determine if you are going with [HTML 5 history mode or hash mode](https://router.vuejs.org/guide/essentials/history-mode.html) It is 2018, and I definitely recommend that you use HTML5 history mode.
+First, you should determine if you are going with [HTML 5 history mode or hash mode](https://router.vuejs.org/guide/essentials/history-mode.html) It is 2020, and we definitely recommend that you use HTML5 history mode.
 
 If you use history mode, then it means your client-side router will need to work in sync with your server-side router.
 
 ## Micro-frontends
 
-[Micro-frontends](https://micro-frontends.org/) is the term you are looking for. Basically, it is your first line of segregation. You should split your app into multiple smaller apps. Each app will have its root component, router, models, services, etc. You will share many of the components (Of course, word very large application is important. And I literally mean it.).
+[Micro-frontends](https://micro-frontends.org/) is the term you are looking for. Basically, it is your first line of segregation. You should split your app into multiple smaller apps. Each app will have it's root component, router, models, services, etc. You will share many of the components (Of course, a very large application is important).
```

Comment:
Use possessive _its_ instead of _it's_ 

in _Each app will have it's root component_

Reply:
done